packages/extras/src/lib/components/Grid/Grid.svelte.d.ts

```diff
@@ -18,7 +18,6 @@ export type GridProps = Props<Mesh> & {
   fadeDistance?: number
   fadeStrength?: number
   side?: Side
-  children?: GridSnippet
```

Comment:
I've noticed a removing the optional `children` prop a couple of times, we didn't need it did we because `children` is a reserved prop right?

Reply:
It's already provided by the threlte `Props` export, so we don't need to explicitly add it unless the children are providing more properties than the standard `ref` 🎉